Web Quiz Engine (Kotlin)/task/src/engine/model/AnswerQuizRequestBody.kt

```diff
@@ -1,3 +1,3 @@
 package engine.model
 
-data class AnswerQuizRequestBody(val answerList: List<Int>?)
+data class AnswerQuizRequestBody(val answer: List<Int>?)
```

Comment:
```suggestion
data class AnswerQuizRequestBody(val answers: List<Int>?)
```

I hate commenting about naming but usually Collection variables are either plural or have "List" or "Map" in the naming.

Reply:
Yep, I know. Except, hyperskill tests fail otherwise as they send payloads with `answer` in them. However, I can use an annotation for that.
And as I abhorr plurals, Done as 
```kotlin
data class AnswerQuizRequestBody(
    @JsonProperty("answer")
    val answerList: List<Int>?,
    )
````